src/PrestaShopBundle/Resources/config/services/core/form/command_builder.yml

```diff
@@ -7,21 +7,30 @@ services:
     arguments:
       - !tagged core.product_command_builder
 
+  prestashop.core.form.command_builder.product.update_product_commands_builder:
```

Comment:
Hi @zuk3975. For new services, plz use the new convention based on FQCN and autowiring. For more details see this ADR : https://github.com/PrestaShop/ADR/blob/master/0019-FQCN-Autowiring.md

Reply:
I guess it's the occasion, and you can also remove all the old ones for this PR this way the whole definition file is clean and we won't need to modify it again in the cleaning PR (because this is the PR that needs to be tested manually no?)